src/test/c_api_tests.cpp

```diff
@@ -236,10 +237,79 @@ TEST_F(CapiInferencePreparationTest, Basic) {
     //////////////////
     //  INFERENCE
     //////////////////
+    // we need to prepare server now TODO
+    // remove when C-API start implemented
+    std::string port = "9000";
+    randomizePort(port);
+    char* argv[] = {
+        (char*)"OpenVINO Model Server",
+        (char*)"--model_name",
+        (char*)"dummy",
+        (char*)"--model_path",
+        (char*)"/ovms/src/test/dummy",
+        (char*)"--port",
+        (char*)port.c_str(),
+        (char*)"--log_level",
+        (char*)"DEBUG",
+        nullptr};
+
+    ovms::Server& server = ovms::Server::instance();
+    std::thread t([&argv, &server]() {
+        ASSERT_EQ(EXIT_SUCCESS, server.start(9, argv));
```

Comment:
Instead of spawning threads, cant we just use non blocking c-api server start here?

Reply:
We could also get rid of CLI params and start it using go/mmo

Reply:
We could also get rid of port randomization

Reply:
start(argc, argv) never happens in real c-api flow

Reply:
This should be already resolved.